projects/mecab/package.yml

```diff
@@ -0,0 +1,42 @@
+distributable:
+  url: https://deb.debian.org/debian/pool/main/m/mecab/mecab_{{version.raw}}.orig.tar.gz
+  # Canonical URL = https://drive.google.com/uc?export=download&id=0B4y35FiV1wh7cENtOXlicTFaRUE
+  # homepage = https://taku910.github.io/mecab/
+  strip-components: 1
+
+versions:
+  - 0.996
+  # Do not think they have versions on the site, homebrew picks it up from deb.debian.org
```

Comment:
@jhheider anywhere you can see we can scrape the versions?

Reply:
in the translated web page, there is a What's New section that has some versions.

Reply:
http://ftp.debian.org/debian/pool/main/m/mecab/

Reply:
good catch, yeah this name is no good. We need an audit that detects such things imo.

Reply:
shouldn't be hard. I envision a tri-partite test
- package name is parsable as a valid URL
- URL.host contains a period
- URL.host is resolvable via DNS